Seasonal Spotlights: Fall article (#1901)
ppy/osu-wiki: news/2018-12-31-beatmap-spotlights-fall-2018.md

﻿---
layout: post
title: "Seasonal Spotlights: Fall 2018"
date: 2018-12-31 06:00:00 +0000
---

> Although the end of 2018 is approaching fast we still have some unopened presents left for this year. Come and take a look at the best maps of the Fall Season and earn a fancy new medal for yourself!

Reviewer: add a comma after "fast"

> The beatmap charts for the Fall Spotlights can be found [here](https://osu.ppy.sh/rankings/osu/charts?spotlight=252). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=chart) for easier access. **The Fall 2018 Spotlights rankings will end on the 15th of March, 2019.**

Reviewer 2: Intro paragraph is first, then the header.

Author: No intro in this version yet

Reviewer 2: Did not read the content of this line, sorry.

Author: intro added with the latest commit

The beatmap charts for the Fall Spotlights can be found [here](https://osu.ppy.sh/rankings/osu/charts?spotlight=252). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=chart) for easier access. **The Fall 2018 Spotlights rankings will end on the 15th of March, 2019.**

## Summer Spotlights results

> First of all, congratulations to the winners of the Seasonal Spotlights: Summer 2018. They receive 3, 2, and 1 months of supporter for 1st, 2nd, and 3rd place respectively for their efforts!

Reviewer: They +will receive 3, 2...

+ **osu!catch:** [RAMPAGE88](https://osu.ppy.sh/users/448547), [Crystal](https://osu.ppy.sh/users/1646397) and [Nelly](https://osu.ppy.sh/users/4741164)
+ **osu!mania:** [\[Crz\]Satori](https://osu.ppy.sh/users/7082178), [robby250](https://osu.ppy.sh/users/2653437) and [Cryolien](https://osu.ppy.sh/users/1626983)

# Navigation

- <a href="#osu">osu!</a>
- <a href="#taiko">osu!taiko</a>
- <a href="#catch">osu!catch</a>
- <a href="#mania">osu!mania</a>

# Fall 2018 Spotlights

## <a name="osu" id="osu"></a>osu!

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/kaoru.jpg)](https://osu.ppy.sh/beatmapsets/397764)

> Contrasting her mapset of "Odoru Mizushibuki", [Hollow Wings](https://osu.ppy.sh/users/416662) brings us a very different interpretion of this melody line in her mapset of the sister arrangement Akiyama Uni's "Kaoru Juyouka". The set is designed to shrink in circle size for each difficulty level, with each difficulty showing off different design concepts for each circle size, cumulating in a wonderful yet challenging CS7 low-bpm stream challenge in the highest difficulty. Careful attention to detail was placed on the stream designs to reflect the various pitches of the melody line, making for a very unique experience nobody should miss.

Reviewer 2: ```suggestion
Contrasting her mapset of "Odoru Mizushibuki", [Hollow Wings](https://osu.ppy.sh/users/416662) brings us a very different interpretation of this melody line in her mapset of the sister arrangement Akiyama Uni's "Kaoru Juyouka". The set is designed to shrink in circle size for each difficulty level, with each difficulty showing off different design concepts for each circle size, accumulating in a wonderful yet challenging CS7 low-bpm stream challenge in the highest difficulty. Careful attention to detail was placed on the stream designs to reflect the various pitches of the melody line, making for a very unique experience nobody should miss.
```

> Contrasting her mapset of "Odoru Mizushibuki", [Hollow Wings](https://osu.ppy.sh/users/416662) brings us a very different interpretation of this melody line in her mapset of the sister arrangement Akiyama Uni's "Kaoru Juyouka". The set is designed to shrink in circle size for each difficulty level, with each difficulty showing off different design concepts for each circle size, accumulating in a wonderful yet challenging Circle Size 7 low-bpm stream challenge in the highest difficulty. Careful attention to detail was placed on the stream designs to reflect the various pitches of the melody line, making for a very unique experience nobody should miss.

Reviewer 3: low-bpm  -> low BPM
on line 121 there is "high BPM", so this should be consistent.

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/abst.jpg)](https://osu.ppy.sh/beatmapsets/839256)

> [Ryuusei Aika](https://osu.ppy.sh/users/7777875), known best for his extremely high BN activity and his clean mapping style, brings forth AcuticNotes' "Abst:Re:Act" in collaboration with [Reincarnation](https://osu.ppy.sh/users/4086593) (Dynamix) and [Suzuki_1112](https://osu.ppy.sh/users/3170678). The map does not let up till the ending slider art, constantly throwing crazy stream and kickslider patterns mixed with extreme spacing changes sure to challenge even the most confident of technical players. If that's too much for you, give the guest difficulty a try - it's easier than the top difficulty, but not by much!

Reviewer 3: Who is Reincarnation and how are they related to this set? The description of the map credits the GD to DJMax who is Dynamix, and who is not Reincarnation.

Author: Probably some stupid renames along the way

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/great-spirit.jpg)](https://osu.ppy.sh/beatmapsets/765801)

> [IceKalt](https://osu.ppy.sh/users/5410645)'s rendition of Armin van Buuren's "Great Spirit" captures the epic feeling of the song extremely well. Be it through expertly arranged short slider patterns or jump patterns with great usage of stop-and-go motions, each of the song's many climaxes is represented in a unique and challenging way. The calm parts are filled with beautiful and varied sliderart, making this map enjoyable all the way through!

Reviewer: sliderart -> slider art

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/hotaru.jpg)](https://osu.ppy.sh/beatmapsets/580596)

> [Narcissu](https://osu.ppy.sh/users/1826598) manages to impress us once again with his map of Fujita Maiko's "Hotaru", bringing clean patterns as well as engaging and varying jumps that manage to perfectly capture the feeling of the song. The use of simpler rhythm lets the map focus on using its patterns to the fullest to complement the song, supported by the well done hitsounding.

Reviewer 4: ```suggestion
[Narcissu](https://osu.ppy.sh/users/1826598) manages to impress us once again with his map of Fujita Maiko's "Hotaru", bringing clean patterns as well as engaging and varying jumps that manage to perfectly capture the feeling of the song. The use of simpler rhythm lets the map focus on using its patterns to the fullest to complement the song, supported by the well-done hitsounding.
```

> [Narcissu](https://osu.ppy.sh/users/1826598) manages to impress us once again with his map of Fujita Maiko's "Hotaru", bringing clean patterns as well as engaging and varying jumps that manage to perfectly capture the feeling of the song. The use of simpler rhythm lets the map focus on using it's patterns to the fullest to complement the song, supported by the well done hitsounding.

Reviewer: "lets the map focus on using it's patterns" -> "using its* patterns"

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/kairikou.jpg)](https://osu.ppy.sh/beatmapsets/832836)

> With his map of Feryquitous feat. Aitsuki Nakuru's "Kairikou", [cococolaco](https://osu.ppy.sh/users/9579526) brings us a highly polished, enjoyable map. It strikes the perfect balance between easy-going patterning and varied emphasis, making it more smooth to play than the average mapset while still representing the song in a detailed way. Dive right in and enjoy how this map and its beautiful song flow by together!

Reviewer: more smooth to play -> smoother to play

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/turii.jpg)](https://osu.ppy.sh/beatmapsets/800638)

> With a different take on an already ranked map, [celerih](https://osu.ppy.sh/users/4696296)'s map of Zektbach's "Turii \~Panta rhei\~" shows us untapped potential that the song has to offer. The map takes a focus on movement through streams, there are lots of twists and turns that aren't seen in most stream maps. All of this combined with a tricky 1/3 section and the low approach rate, the map becomes a fun challenge for the casual and experienced player. Think the Insane difficulty is too easy? Play [Muya](https://osu.ppy.sh/users/153323)'s diff for a real challenge.

Reviewer 4: ```suggestion
With a different take on an already ranked map, [celerih](https://osu.ppy.sh/users/4696296)'s map of Zektbach's "Turii \~Panta rhei\~" shows us untapped potential that the song has to offer. The map takes a focus on movement through streams; there are lots of twists and turns that aren't seen in most stream maps. All of this combined with a tricky 1/3 section and the low approach rate, and the map becomes a fun challenge for the casual and experienced player. Think the Insane difficulty is too easy? Play [Muya](https://osu.ppy.sh/users/153323)'s difficulty for a real challenge.
```

> With a different take on an already ranked map, [celerih](https://osu.ppy.sh/users/4696296)'s map of Zektbach's "Turii ~Panta rhei~" shows us untapped potential that the song has to offer. The map takes a focus on movement through streams, there's lots of twists in turns that aren't seen in most stream maps. All of this combined with a tricky 1/3 section and the low approach rate, the map becomes a fun challenge for the casual and experienced player. Think the Insane difficulty is too easy? Play [Muya](https://osu.ppy.sh/users/153323)'s diff for a real challenge.

Reviewer: there's lots of twists -> there're OR there are lots of twists

> With a different take on an already ranked map, [celerih](https://osu.ppy.sh/users/4696296)'s map of Zektbach's "Turii ~Panta rhei~" shows us untapped potential that the song has to offer. The map takes a focus on movement through streams, there's lots of twists in turns that aren't seen in most stream maps. All of this combined with a tricky 1/3 section and the low approach rate, the map becomes a fun challenge for the casual and experienced player. Think the Insane difficulty is too easy? Play [Muya](https://osu.ppy.sh/users/153323)'s diff for a real challenge.

Reviewer 5: You need to escape the "~" in Panta rhei -> `\~Panta rhei\~`

> With a different take on an already ranked map, [celerih](https://osu.ppy.sh/users/4696296)'s map of Zektbach's "Turii ~Panta rhei~" shows us untapped potential that the song has to offer. The map takes a focus on movement through streams, there's lots of twists in turns that aren't seen in most stream maps. All of this combined with a tricky 1/3 section and the low approach rate, the map becomes a fun challenge for the casual and experienced player. Think the Insane difficulty is too easy? Play [Muya](https://osu.ppy.sh/users/153323)'s diff for a real challenge.

Reviewer 3: twists in turns -> twists and turns
The common phrase is the latter, never heard of the former before

[Uberzolik](https://osu.ppy.sh/users/1314547) may be new to the mapping scene, but their map of Chata's "Labyrinth" shows just how promising they are. The map captures the relaxing tune by mixing in sharp and wide angles for emphasis, and a focus on mapping the calm vocals when they are present. Together this makes for a very relaxing yet rewarding experience without going overboard. So sit down, and relax to Chata's cute and soothing voice.

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/poppy.jpg)](https://osu.ppy.sh/beatmapsets/838383)

Featuring a song from the quirky Internet star Poppy's "I'm Poppy", [Myxomatosis](https://osu.ppy.sh/users/2202645) and [Mao](https://osu.ppy.sh/users/2204515) give us this wonderful set brimming with energy and personality. Every difficulty manages to perfectly capture the robotic nature of the song, without feeling stiff or unnatural. Good luck trying to play this map without a smile. Follow Poppy's steps and enjoy this fantastic mapset.

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/utsukushii.jpg)](https://osu.ppy.sh/beatmapsets/777597)

> [dsco](https://osu.ppy.sh/users/4330511) wows us again with their unusual mapset of the math rock ballad Paranoid Void's “Utsukushii Kodoku”. [dsco](https://osu.ppy.sh/users/4330511)'s top diff, Escapism, creates contrast between the dense, chaotic instrumental solos and the fluctuating vocal sections with its creative use of flow, movement, and overlaps, as well as doing an expert job of challenging your reading and rhythmic skills. For players who aren't ready for Escapism's unique challenge, [taku](https://osu.ppy.sh/users/684433)’s difficulty has some equally interesting patterning and movement, and dsco's lower diffs are interesting in their own right. Make sure to give this set a try for an unconventional fresh experience.

Reviewer: diffs -> difficulties

> [dsco](https://osu.ppy.sh/users/4330511) wows us again with their unusual mapset of the math rock ballad Paranoid Void's “Utsukushii Kodoku”. [dsco](https://osu.ppy.sh/users/4330511)'s top difficulty, Escapism, creates contrast between the dense, chaotic instrumental solos and the fluctuating vocal sections with its creative use of flow, movement, and overlaps, as well as doing an expert job of challenging your reading and rhythmic skills. For players who aren't ready for Escapism's unique challenge, [taku](https://osu.ppy.sh/users/684433)’s difficulty has some equally interesting patterning and movement, and dsco's lower difficulties are interesting in their own right. Make sure to give this set a try for an unconventional fresh experience.

Reviewer 4: ```suggestion
[dsco](https://osu.ppy.sh/users/4330511) wows us again with their unusual mapset of the math rock ballad that is paranoid void's “Utsukushii Kodoku”. [dsco](https://osu.ppy.sh/users/4330511)'s top difficulty, Escapism, creates contrast between the dense, chaotic instrumental solos and the fluctuating vocal sections with its creative use of flow, movement, and overlaps, as well as doing an expert job of challenging your reading and rhythmic skills. For players who aren't ready for Escapism's unique challenge, [taku](https://osu.ppy.sh/users/684433)’s difficulty has some equally interesting patterning and movement, and dsco's lower difficulties are interesting in their own right. Make sure to give this set a try for an unconventional fresh experience.
```

> [Net0](https://osu.ppy.sh/users/5099768) beautifully portrays the story behind Utsu-P's "Uchuujin no I Love You" through his various strucutral and rhythm choices, leaving the player with a bizzare, yet unique play experience. His combo colors and storyboard visuals allign together beautifully to help tell the story. That, combined with his various contrasts in intensity and use of repetitive slider shapes during the chorus shows a close attention to detail. If you think you have an eye for song representation and thought provoking maps, sit back and enjoy [Net0](https://osu.ppy.sh/users/5099768)'s Uchuujin no I Love You!

Reviewer 2: ```suggestion
[Net0](https://osu.ppy.sh/users/5099768) beautifully portrays the story behind Utsu-P's "Uchuujin no I Love You" through his various structural and rhythm choices, leaving the player with a bizarre, yet unique play experience. His combo colors and storyboard visuals align together beautifully to help tell the story. That, combined with his various contrasts in intensity and use of repetitive slider shapes during the chorus shows a close attention to detail. If you think you have an eye for song representation and thought provoking maps, sit back and enjoy [Net0](https://osu.ppy.sh/users/5099768)'s Uchuujin no I Love You!
```

> [Net0](https://osu.ppy.sh/users/5099768) beautifully portrays the story behind Utsu-P's "Uchuujin no I Love You" through his various strucutral and rhythm choices, leaving the player with a bizzare, yet unique play experience. His combo colors and storyboard visuals align together beautifully to help tell the story. That, combined with his various contrasts in intensity and use of repetitive slider shapes during the chorus shows a close attention to detail. If you think you have an eye for song representation and thought provoking maps, sit back and enjoy [Net0](https://osu.ppy.sh/users/5099768)'s Uchuujin no I Love You!

Reviewer 3: thought provoking -> thought-provoking
it should be a hyphenated term

Storyboard is made by me (Noffy) with the base ideas given by Net0 , current description makes it sound like Net0 did it all himself :(

> [Net0](https://osu.ppy.sh/users/5099768) beautifully portrays the story behind Utsu-P's "Uchuujin no I Love You" through his various strucutral and rhythm choices, leaving the player with a bizzare, yet unique play experience. His combo colors and [Noffy](https://osu.ppy.sh/users/1541323)'s storyboard visuals align together beautifully to help tell the story. That, combined with his various contrasts in intensity and use of repetitive slider shapes during the chorus shows a close attention to detail. If you think you have an eye for song representation and thought-provoking maps, sit back and enjoy [Net0](https://osu.ppy.sh/users/5099768)'s Uchuujin no I Love You!

Reviewer 2: `strucutral` -> `structural`
`bizzare` -> `bizarre`

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/kimi-no-gin.jpg)](https://osu.ppy.sh/beatmapsets/876259)

> [Bearizm](https://osu.ppy.sh/users/3495259) kicks off the season with his return to the mapping scene after a year-long hiatus. His interpretation of Kalafina's "Kimi no Gin no Niwa" is one of the more expressive beatmaps in the game simply due to its use of dynamic slider velocity changes that follows the intensity of the vocals and violins, as well as having section emphasis, meaning that every chorus has a different set of rhythms and emphasis to show progression and a variety of different pacing and movements. This not only helps create variety throughout the song, but it compliments the dynamic slider velocity changes that the map offers because players will constantly need to pay full attention to momentum changes.

Reviewer: one of the more -> one of the most*

> [Bearizm](https://osu.ppy.sh/users/3495259) kicks off the season with his return to the mapping scene after a year-long hiatus. His interpretation of Kalafina's "Kimi no Gin no Niwa" is one of the more expressive beatmaps in the game simply due to its use of dynamic slider velocity changes that follows the intensity of the vocals and violins, as well as having section emphasis, meaning that every chorus has a different set of rhythms and emphasis to show progression and a variety of different pacing and movements. This not only helps create variety throughout the song, but it compliments the dynamic slider velocity changes that the map offers because players will constantly need to pay full attention to momentum changes.

Reviewer: "emphasis" is overutilized and redundant, try "set of rhythms and interpretations" or something of the sorts

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/wakusei.jpg)](https://osu.ppy.sh/beatmapsets/841634)

Marking an older mapper's return to the mapping scene, [gowww](https://osu.ppy.sh/users/119404)'s map of HACHI's "Suna no Wakusei" is nothing but a blast to play. His pattern-based mapping style and unconventional emphasis aren't intuitive for everyone, but engaging more with it will make this map stand out from the rest. The Extra difficulty keeps you on your toes throughout the whole map with a focus on different difficulty elements in each part, while finally climaxing into a straining jump section in the bridge with improvised emphasis. Along with well-made lower diffs that also differ from the norm, this mapset is a very unique and memorable experience.

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/musique.jpg)](https://osu.ppy.sh/beatmapsets/816542)

> Fal's "ame no musique" is a super relaxing piece of music with a lot of rhythmic detail, and [LMT](https://osu.ppy.sh/users/7262798)'s rendition of it captures this feeling perfectly. The flow is smooth and the difficulty is fairly low, but the piano and string fills throughout the song are mapped in a very detailed way, making the rhythms throughout the map extremely varied and creating a unique challenge when played with difficulty increasing mods. Please try it our yourself!

Reviewer: "throughout" is repeated, try another word such as "making the rhythms of/during the map"

> Fal's "ame no musique" is a super relaxing piece of music with a lot of rhythmic detail, and [LMT](https://osu.ppy.sh/users/7262798)'s rendition of it captures this feeling perfectly. The flow is smooth and the difficulty is fairly low, but the piano and string fills throughout the song are mapped in a very detailed way, making the rhythms throughout the map extremely varied and creating a unique challenge when played with difficulty increasing mods. Please try it our yourself!

Reviewer: try it our -> try it out*

> [Rolniczy](https://osu.ppy.sh/users/8331132) is a newer mapper, only ranking his first map in 2018 but doesn't fail to keep innovating and changing how his maps are built and felt. In Kalafina's "Manten" he uses a lot of extended sliders to follow the vocals very closely while still keeping a lot of attention to the movement with its direction changes. In parts where the vocals drop away, it switches to more circle heavy style with wider angles and stops to mark clear gaps in the intrumentals. This all makes for a unique experience not many other maps give.

Reviewer 2: ```suggestion
[Rolniczy](https://osu.ppy.sh/users/8331132) is a newer mapper, only ranking his first map in 2018 but doesn't fail to keep innovating and changing how his maps are built and felt. In Kalafina's "Manten" he uses a lot of extended sliders to follow the vocals very closely while still keeping a lot of attention to the movement with its direction changes. In parts where the vocals drop away, it switches to more circle heavy style with wider angles and stops to mark clear gaps in the instrumentals. This all makes for a unique experience not many other maps give.
```

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/dark-water.jpg)](https://osu.ppy.sh/beatmapsets/761140)

> It's rare to find a mapper debut as impressive as this one, and its even more rare that one will give you goosebumps. [Godot](https://osu.ppy.sh/users/6466477) manages to tie his map beautifully to Agent Fresco's "Dark Water" through his unique usage of repetitive slider rhythms, complimented by rotating drum patterns that are fun to play. Complimented with a Hard difficulty that's also concept based, there's something in this set for everybody. With such an enticing map as this one, we hope to see more from [Godot](https://osu.ppy.sh/users/6466477) in the ranked section in the future!

Reviewer 2: ```suggestion
It's rare to find a mapper debut as impressive as this one, and it's even more rare that one will give you goosebumps. [Godot](https://osu.ppy.sh/users/6466477) manages to tie his map beautifully to Agent Fresco's "Dark Water" through his unique usage of repetitive slider rhythms, complemented by rotating drum patterns that are fun to play. Complemented with a Hard difficulty that's also concept based, there's something in this set for everybody. With such an enticing map as this one, we hope to see more from [Godot](https://osu.ppy.sh/users/6466477) in the ranked section in the future!
```

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/triangle.jpg)](https://osu.ppy.sh/beatmapsets/862156)

Touhou lovers, this one's for you. Liz Triangle's "With Triangle" is [celerih](https://osu.ppy.sh/users/4696296)'s wonderful tribute to the circle Liz Triangle, celebrating their past 9 years of music. This map is an amazing yet soothing experience that shows celerih's expert use of simple yet very effective patterns to express the various songs' emotions while playing effortlessly. [Hailie](https://osu.ppy.sh/users/5226970)'s hitsounds complement the map very well and give it a very nice touch, so make sure to pay attention to those! The map features a simple but clean storyboard giving info and different backgrounds for every song included, making the map feel as personal as possible. A must play, even if Touhou isn't your cup of tea!

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/blanche.jpg)](https://osu.ppy.sh/beatmapsets/759524)

> Dormin's "Une mage blanche" shows [Sulfur](https://osu.ppy.sh/users/5297447)'s expertise of movement and flow throughout the Insane diff very well and makes this map a blast to play alongside neat usage of default sample hitsounds, giving the map a nice classic feel. However, don't overestimate this map as it will test your finger control with its frequent use of triples and 3/4 sliders combined with circle heavy sections. If you're still looking for a bigger challenge, go ahead and try out [Kalibe](https://osu.ppy.sh/users/3376777)'s awesome Extra which will challenge your aim even harder while keeping that same effortless movement through Russian inspired patterns!

Reviewer: diff -> difficulty

> Dormin's "Une mage blanche" shows [Sulfur](https://osu.ppy.sh/users/5297447)'s expertise of movement and flow throughout the Insane diff very well and makes this map a blast to play alongside neat usage of default sample hitsounds, giving the map a nice classic feel. However, don't overestimate this map as it will test your finger control with its frequent use of triples and 3/4 sliders combined with circle heavy sections. If you're still looking for a bigger challenge, go ahead and try out [Kalibe](https://osu.ppy.sh/users/3376777)'s awesome Extra which will challenge your aim even harder while keeping that same effortless movement through Russian inspired patterns!

Reviewer: don't overestimate -> don't underestimate

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/good-night.jpg)](https://osu.ppy.sh/beatmapsets/679935)

> [Chaoslitz](https://osu.ppy.sh/users/3621552) and Moecho’s collab on colate and Nanahira’s "Good Night" is a great example of how collab difficulties are supposed to be mapped: the styles of both mappers blending with each other, resulting in a smooth playing experience without either of the mappers’ parts outshining the other. Coupled with the intuitive rhythm and patterns, the result is a very enjoyable map to play for players of all skillsets and levels.

Reviewer: this description is the most generic and uninspired I have ever seen, but oh well!

Author: ¯\_(ツ)_/¯

## <a name="taiko" id="taiko"></a>osu!taiko

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/passage.jpg)](http://osu.ppy.sh/beatmapsets/587426)

The winning mapset of this season's beatmap Spotlight is nothing less than The Flashbulb's "Passage D", mapped by [Raiden](https://osu.ppy.sh/users/2239480)!

> A song known across several games and game modes, it has found its way into osu!taiko's ranked section a few months after being featured as a map in the 2018 osu!taiko World Cup Finals. Coincidence? Who knows. What I do know is that this entire map set is incredibly well-mapped, which is a big accomplishment given the calm yet chaotic nature of this song. 

Reviewer 4: ```suggestion
A song known across several games and game modes, it has found its way into osu!taiko's ranked section a few months after being featured as a map in the osu!taiko World Cup 2018 Finals. Coincidence? Who knows. What I do know is that this entire map set is incredibly well-mapped, which is a big accomplishment given the calm yet chaotic nature of this song. 
```

> Raiden has managed to make an Ura Oni (named Raidcore, a difficulty name he uses for many of his high-difficulty maps) which balances playability and a huge amount of various snaps very well. It's not uncommon to find 4 different snaps in one chain of 1/4 notes, the common snapping. This is difficult to read, difficult to play, and difficult to map well. And yet, it follows the song very nicely, its flow (and "anti-flow") works, and that extends to all difficulties.

Reviewer: link to profile on Raiden

Author: Only the first mention of the username needs to be hyperlinked. That happened during the first sentence

> Raiden has managed to make an Ura Oni (named Raidcore, a difficulty name he uses for many of his high-difficulty maps) which balances playability and a huge amount of various snaps very well. It's not uncommon to find 4 different snaps in one chain of 1/4 notes, the common snapping. This is difficult to read, difficult to play, and difficult to map well. And yet, it follows the song very nicely, its flow (and "anti-flow") works, and that extends to all difficulties.

Reviewer 3: "It's not uncommon to find 4 different snaps in one chain of 1/4 notes, the common snapping."

what? how would you find different snaps if it's specifically in a chain of 1/4? this is a bit confusing the way it's currently written.

Reviewer 3: Even "It's not uncommon to find 4 different snaps mixed in with a chain of 1/4 notes, the common snapping." would be better

> Raiden has managed to make an Ura Oni (named Raidcore, a difficulty name he uses for many of his high-difficulty maps) which balances playability and a huge amount of various snaps very well. It's not uncommon to find 4 different snaps mixed in with a chain of 1/4 notes, the common snapping. This is difficult to read, difficult to play, and difficult to map well. And yet, it follows the song very nicely, its flow (and "anti-flow") works, and that extends to all difficulties.

Reviewer 4: ```suggestion
Raiden has managed to make an Ura Oni (named Raidcore, a difficulty name he uses for many of his high-difficulty maps) which balances play-ability and a huge amount of various snaps very well. It's not uncommon to find 4 different snaps mixed in with a chain of 1/4 notes, the common snapping. This is difficult to read, difficult to play, and difficult to map well—and yet, it follows the song very nicely; its flow (and "anti-flow") works, and that extends to all difficulties.
```

> A very well-deserved #1 spot for this season's Spotlight!

Reviewer: change to 1st instead to avoid markdown issues?

> Our runner-up mapset for this season, Chroma's "Sayonara Planet Wars" single-handedly mapped by [Greenshell](https://osu.ppy.sh/users/8693851) sure was a strong contender. Featuring one of the best (and nowadays, quite rare to find) lower diffs in the sense that, while following the current criterias, it doesnt lose its identity at all, to the point you could recognize what song is playing behind this rather unique mapping.

Reviewer 2: ```suggestion
Our runner-up mapset for this season, Chroma's "Sayonara Planet Wars" single-handedly mapped by [Greenshell](https://osu.ppy.sh/users/8693851) sure was a strong contender. Featuring one of the best (and nowadays, quite rare to find) lower diffs in the sense that, while following the current criterias, it doesn't lose its identity at all, to the point you could recognize what song is playing behind this rather unique mapping.
```

> Our runner-up mapset for this season, Chroma's "Sayonara Planet Wars" single-handedly mapped by [Greenshell](https://osu.ppy.sh/users/8693851) sure was a strong contender. Featuring one of the best (and nowadays, quite rare to find) lower difficulties in the sense that, while following the current criterias, it doesn't lose its identity at all, to the point you could recognize what song is playing behind this rather unique mapping.

Reviewer 3: criterias -> criteria
criteria is plural already.

> In the other hand, we got this really fun top diff that stands in here to tell us that, in a world ruled by techy 1/4+1/6 maps and high BPM speed maps, stream maps can still be quite fun and a worthy challenge even for the best of us.

Reviewer: On the other hand only works if you used "on one hand" previously. Change to "On a side note / In other news / In addition"

> In the other hand, we got this really fun top diff that stands in here to tell us that, in a world ruled by techy 1/4+1/6 maps and high BPM speed maps, stream maps can still be quite fun and a worthy challenge even for the best of us.

Reviewer: diff -> difficulty

> In the other hand, we got this really fun top diff that stands in here to tell us that, in a world ruled by techy 1/4+1/6 maps and high BPM speed maps, stream maps can still be quite fun and a worthy challenge even for the best of us.

Reviewer: we got -> we've got

> In the other hand, we got this really fun top diff that stands in here to tell us that, in a world ruled by techy 1/4+1/6 maps and high BPM speed maps, stream maps can still be quite fun and a worthy challenge even for the best of us.

Reviewer: high BPM speed maps is redundant, either high BPM or speed maps, choose one

> In addition, we've got this really fun top difficulty that stands in here to tell us that, in a world ruled by techy 1/4+1/6 maps and speed maps, stream maps can still be quite fun and a worthy challenge even for the best of us.

Reviewer 4: combine this line with the next

> This entire mapset holds quite its value in there and we hope all of you can get a pleasant time while trying to get some points out of this set!

Reviewer 4: ```suggestion
This entire mapset holds quite its value in there and we hope all of you can have a pleasant time while trying to get some points out of this set!
```

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/sound-chimera.jpg)](http://osu.ppy.sh/beatmapsets/779916)

> [yea](https://osu.ppy.sh/users/6874332)’s map of Laur's “Sound Chimera” comes in a close 3rd place for this quarter’s spotlights featuring a ton of finishers and some groovy slider velocity usage! Being the first marathon out of many for this round of spotlights, it certainly stands out in its own unique way. This map is truly a finished product, with Laur’s signature diversity in music and yea’s impeccable translation of that through intuitive finisher usage, slider velocity and density balance.

Reviewer: profile link on "yea's impeccable..."

> [yea](https://osu.ppy.sh/users/6874332)’s map of Laur's “Sound Chimera” comes in a close 3rd place for this quarter’s spotlights featuring a ton of finishers and some groovy slider velocity usage! Being the first marathon out of many for this round of spotlights, it certainly stands out in its own unique way. This map is truly a finished product, with Laur’s signature diversity in music and yea’s impeccable translation of that through intuitive finisher usage, slider velocity and density balance.

Reviewer 4: ```suggestion
[yea](https://osu.ppy.sh/users/6874332)’s map of Laur's “Sound Chimera” comes in a close 3rd place for this quarter’s Spotlights, featuring a ton of finishers and some groovy slider velocity usage! Being the first marathon out of many for this round of Spotlights, it certainly stands out in its own unique way. This map is truly a finished product, with Laur’s signature diversity in music and yea’s impeccable translation of that through intuitive finisher usage, slider velocity and density balance.
```

> yea is relatively new to ranking maps, but currently has 5 loved maps ranging from 7 to 9 stars, so if you enjoy a challenge, head over to his profile. This truly is just a fun map, and I am not doing it any justice trying to sit here explaining why it’s so good, so just go and give it a play to find out for yourself!

Reviewer: profile link on yea

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/last-minute.jpg)](http://osu.ppy.sh/beatmapsets/835011)

A pp map in spotlights? No way!

> [\[Zeth\]](https://osu.ppy.sh/users/9912966)’s map of sanj's “last minute (sakuraburst remix)” breaks the mold with subtle but tasteful sv, a complex and interesting song, 1/8 doubles like you’ve never seen and pp for only deserving parties. Marathons maps often stand out due to their ability to change pace and texture several times during one map, and boy, does \[Zeth] sure capitalize on that.

Reviewer: tasteful sv -> tasteful slider velocity changes/manipulation

Reviewer: missing second profile link on "does [Zeth] sure capitalize on that"

> [\[Zeth\]](https://osu.ppy.sh/users/9912966)’s map of sanj's “last minute (sakuraburst remix)” breaks the mold with subtle but tasteful slider velocity changes, a complex and interesting song, 1/8 doubles like you’ve never seen and pp for only deserving parties. Marathons maps often stand out due to their ability to change pace and texture several times during one map, and boy, does \[Zeth] sure capitalize on that.

Reviewer 4: ```suggestion
[\[Zeth\]](https://osu.ppy.sh/users/9912966)’s map of sanj's “last minute (sakuraburst remix)” breaks the mold with subtle but tasteful slider velocity changes, a complex and interesting song, 1/8 doubles like you’ve never seen and pp for only deserving parties. Marathons maps often stand out due to their ability to change pace and texture several times during one map, and boy, does \[Zeth\] sure capitalize on that.
```

The use of 1/8 doubles in this map is what makes it stand out among other similar marathons, since these make sense in the context of the music. With the 1/6 in the kiais leading up to them, they make for a great intensity boost, and are prolonged for such a time that separates the goods from the greats when it comes down to getting a full combo.

> With this being \[Zeth]’s first and only ranked map so far, only time can tell what will come of this talented individual.

Reviewer: missing profile link again

> With this being \[Zeth]’s first and only ranked map so far, only time can tell what will come of this talented individual.

Reviewer 4: ```suggestion
With this being \[Zeth\]’s first and only ranked map so far, only time can tell what will come of this talented individual.
```

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/egret.jpg)](http://osu.ppy.sh/beatmapsets/821745)

> Holding quite the catalogue of ranked maps in the Spotlights section, [Nifty](https://osu.ppy.sh/users/4956097) expands on it with another beautiful mapset, this time with the song Dirty Androids' "Egret and Willow", and boy is this a good one!

Reviewer: "with the song "Egret and Willow" from Dirty Androids" sounds much better

Author: Removed "the song" instead. Prefer to have consistency on artist and title formatting

> The main aspects of this set are a well-formed spread of lower difficulties to appease players on all skill levels, and an expertly crafted highest difficulty (Ebb and Flow), which takes you on a real trip: to perfect stream pattern consistency, to clever usage of Slider Velocity creating an "ebb and flow" visual effect, to unique ways of arranging 1/6 patterns within 1/4 streams which combine creativity and playability in great synergy, to high finisher density at just the right moment - this map covers all the song's subtleties and beautifully converts them into both a visual treat and an incredibly fun finger twister!

Reviewer: on all skill levels -> of* all skill levels

Combine all of these things, and you get a mapset that takes a more than deserved place in this season's Spotlights!

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/eastern-dream.jpg)](http://osu.ppy.sh/beatmapsets/776165)

[ASPIRIN](https://osu.ppy.sh/users/6487115)’s map of Demetori's “Kourou ~ Eastern Dream” joins the spotlights as one of the few and far between ranked maps over 10 minutes!

> However, ASPIRIN has proved that size doesn’t matter, since this map will keep you engaged from its soothing guitar melodies to the finisher-filled, polyrhythmic kiai sections. ASPIRIN does an amazing job at conveying the constantly changing energy of the music in every section. 

Reviewer: missing profile link on ASPIRIN

> The map feels considerably shorter than every other 10+ minute map, such as Nishizumi’s worldenddominator or Midnaait’s International OTOMAD’s 6. Even if you are not typically a fan of Demetori, I suggest you give this map a try, as it is a journey you will not forget.

Reviewer: avoid comparisons to other maps, this is to spotlight this map, not to belittle others

> [![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/icaros.jpg)](http://osu.ppy.sh/beatmapsets/762660)

Reviewer 4: name change in both the image and description

Author: Oh no, I do not care. First, that name seems like a joke which will be reverted soon enough. Second, he made the changes after I made the images and the article should have been released already. 99% of the player base knows his previous name anyway.

Author: It is a pain in the ass to change something on the images after they are done. It might just be me, but I rather not spend that time for this kind of stupidity.

[ARGENTINE DREAM](https://osu.ppy.sh/users/1981781) is no outsider to metal songs, and she gracefully demonstrates it with this new masterpiece on Emperor's "An Elegy Of Icaros".

The unpredictability of the song does not stop her from masterfully composing a blend of long streams, intercalated 1/6 patterns, shameless usage of finishers and a very intelligent representation of specific musical concepts such as guitar tremolos and percussion flams, which are mapped as 1/8 doubles. Despite the song being repeated structurally, the sheer variation of meters (5/4-3/4 to simple 4/4), BPM and snaps add a vast amount of spice to an already spicy mix.

> In short, one of the best if not the best black metal maps to exist for now in osu!taiko.

Reviewer 4: ```suggestion
In short, this is one of the best (if not the best) black metal maps to exist for now in osu!taiko.
```

Reviewer: change to "one of if not the best"

> In short, one of the best if not the best black metal maps to exist for now.

Reviewer: add "in osu!taiko" after "for now"

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/161914.jpg)](http://osu.ppy.sh/beatmapsets/832971)

"Unusual" would be my word choice to describe this set. Not only the song, which is a fairly uncommon eerie breakcore one, but also the song's title and artist, Sorry about my face's "161914". What do they mean? Will we ever know?

Another unusual (but good!) trait of this set by [vrnl](https://osu.ppy.sh/users/4799788) is its ability to stay almost perfectly linear progression-wise while retaining the core aspects of the song, and this is, as you may imagine, not an easy task in a breakcore song. The highest difficulty manages to squeeze every single element the song has to offer, with 1/6 patterns reaching the astounding amount of 13 notes in a row!

> Overall, despite possibly needing some more spice in terms of slider velocity manipulation, the set accomplishes its objectives quite well: linear progression and challenging top difficulty. A well-deserved place in this season's Spotlights!

Reviewer 4: ```suggestion
Overall, despite possibly needing some more spice in terms of slider velocity manipulation, the set accomplishes its objectives quite well: linear progression and challenge in the top difficulty. This map will take its well-deserved place in this season's Spotlights!
```

Reviewer: don't think these are necessary

Author: Yeah, this is fine in my opinion too.

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/border.jpg)](http://osu.ppy.sh/beatmapsets/732190)

> This great set by [\-Kazu-](https://osu.ppy.sh/users/920861) exemplifies what puts him head and shoulders above the competition. Extremely fun top diff that is very difficult but fair, this map harkens back to the very old days of IOSYS in its difficulty/pattern homages.

Reviewer 4: ```suggestion
This great set by [-Kazu-](https://osu.ppy.sh/users/920861) exemplifies what puts him head and shoulders above the competition. With an extremely fun top diff that is very difficult but fair, this map harkens back to the very old days of IOSYS in its difficulty and pattern homages.
```

> This great set by [\-Kazu-](https://osu.ppy.sh/users/920861) exemplifies what puts him head and shoulders above the competition. Extremely fun top diff that is very difficult but fair, this map harkens back to the very old days of IOSYS in it's difficulty/pattern homages.

Reviewer 3: in it's difficulty/pattern homages. -> in its difficulty/pattern homages.

> Kazu's often abstract mapping style really works for this kind of abrasive music, and generally we feel this map has great variation across all difficulties which allows you to appreciate all textures of the music. The lower difficulties maintain their identity without copying elements from the higher ones, and they all have distinct sections that players can appreciate.

Reviewer: link to profile

Author: They are only hyperlinked once per comment, This is done in the previous paragraph.

Truly, we won't just be on the "Border of extacy", we'll be thoroughly engulfed in it while playing this map!

## <a name="catch" id="catch"></a>osu!catch

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/misanthrope.jpg)](http://osu.ppy.sh/beatmapsets/797236)

[Deif](https://osu.ppy.sh/users/318565) comes back with yet another technical masterpiece that pushes the boundaries of pattern usage on LeaF's "Alice in Misanthrope -Ensei Alice-". All difficulties are amazingly mapped and keep a high tension during the energetic song. In the highest difficulty, even a small error in platter placement will lead to a miss. Even in the lower difficulties, platter control is emphasized, giving players a refreshing gameplay experience.

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/salvation.jpg)](http://osu.ppy.sh/beatmapsets/840189)

> One of the most well-known mappers, [alienflybot](https://osu.ppy.sh/users/636114), once again brought excellence as well as challenges to fruit-enthusiaists, with his new marathon map on Yooh's "salvation". Combining aesthetics with intensive hyperdashes, anti-flows and occasional edge jumps, the map well expresses how this epic song goes from calm to hectic. If you are confident about your stamina and precision, this map would surely be the one you don't want to miss!

Reviewer 2: `enthusiaists` -> `enthusiasts`

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/heron.jpg)](http://osu.ppy.sh/beatmapsets/576786)

S-C-U's "heron" is one of the most fun maps you can play if you enjoy catch! This set by [ZiRoX](https://osu.ppy.sh/users/200768), with a guest difficulty by [JBHyperion](https://osu.ppy.sh/users/4879508), has uncommon patterning and interesting mapping. The map fits perfectly with the song, is really enjoyable and challenging for anyone who plays it. If you want something new to check out, try this map and you'll surely enjoy it!

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/wobble-impact.jpg)](http://osu.ppy.sh/beatmapsets/539769)

An incredibly challenging set to play: Sota Fujimori's "Wobble Impact" is a song that never lets up with its pace. Constantly challenging the player with almost no breathing room. [Minato Yukina](https://osu.ppy.sh/users/531253)'s mapping will keep you on your toes as a wide variety of patterns are used to express the diverse set of wubs in the music.

> [![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/blonde.jpg)](http://osu.ppy.sh/beatmapsets/715066)

Reviewer 2: Broken image.
```suggestion
[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/blonde-girl.jpg)](http://osu.ppy.sh/beatmapsets/715066)
```

> If you like energetiic songs, then this is your perfect fit! JENNY ROM's "WWW.BLONDE GIRL" mapped by [Ascendance](https://osu.ppy.sh/users/2931883) brings the upbeaet vibe with the song. With multiple guest difficulties, players can enjoy multiple interpretations of the song. On a side note, make sure to check out the background on Spec's Challenge!

Reviewer 2: ```suggestion
If you like energetic songs, then this is your perfect fit! JENNY ROM's "WWW.BLONDE GIRL" mapped by [Ascendance](https://osu.ppy.sh/users/2931883) brings the upbeat vibe with the song. With multiple guest difficulties, players can enjoy multiple interpretations of the song. On a side note, make sure to check out the background on Spec's Challenge!
```
```suggestion
If you like energetiic songs, then this is your perfect fit! JENNY ROM's "WWW.BLONDE GIRL" mapped by [Ascendance](https://osu.ppy.sh/users/2931883) brings the upbeaet vibe with the song. With multiple guest difficulties, players can enjoy multiple interpretations of the song. On a side note, make sure to check out the background on Spec's Challenge!
```

> If you like energetic songs, then this is your perfect fit! JENNY ROM's "WWW<i></i>&#46;BLONDE GIRL" mapped by [Ascendance](https://osu.ppy.sh/users/2931883) brings the upbeat vibe with the song. With multiple guest difficulties, players can enjoy multiple interpretations of the song. On a side note, make sure to check out the background on Spec's Challenge!

Author: That will become an url that way. We just avoided that with the current text

> If you like energetic songs, then this is your perfect fit! JENNY ROM's "WWW<i></i>&#46;BLONDE GIRL" mapped by [Ascendance](https://osu.ppy.sh/users/2931883) brings the upbeat vibe with the song. With multiple guest difficulties, players can enjoy multiple interpretations of the song. On a side note, make sure to check out the background on Spec's Challenge!

Reviewer 4: that shouldn't be the behavior of strings like this, where did you see it become a link?

> If you like energetic songs, then this is your perfect fit! JENNY ROM's "WWW<i></i>&#46;BLONDE GIRL" mapped by [Ascendance](https://osu.ppy.sh/users/2931883) brings the upbeat vibe with the song. With multiple guest difficulties, players can enjoy multiple interpretations of the song. On a side note, make sure to check out the background on Spec's Challenge!

Author: But it is. Just view the article here with that (altough will look fine currently)   Pointed out previously by Walter by the way

> If you like energetic songs, then this is your perfect fit! JENNY ROM's "WWW<i></i>&#46;BLONDE GIRL" mapped by [Ascendance](https://osu.ppy.sh/users/2931883) brings the upbeat vibe with the song. With multiple guest difficulties, players can enjoy multiple interpretations of the song. On a side note, make sure to check out the background on Spec's Challenge!

Author: Hopefully, you are right. I won't change this again if not.

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/dearest.jpg)](http://osu.ppy.sh/beatmapsets/842871)

> The ultimate duo, consisting of [Ascendance](https://osu.ppy.sh/users/2931883) and [Fii](https://osu.ppy.sh/users/3922569), is back once again with their rendition of Supercell's "My Dearest". Featuring an impressive storyboard by Storyboarder and PoNo, the map is quite the experience and manages to keep bringing fun and engaging patterns for it's entire duration, making the map fly by. Make sure to check this map out, as it is a definite highlight when it comes to these two mappers.

Reviewer 2: ```suggestion
The ultimate duo, consisting of [Ascendance](https://osu.ppy.sh/users/2931883) and [Fii](https://osu.ppy.sh/users/3922569), is back once again with their rendition of Supercell's "My Dearest". Featuring an impressive storyboard by Storyboarder and PoNo, the map is quite the experience and manages to keep bringing fun and engaging patterns for its entire duration, making the map fly by. Make sure to check this map out, as it is a definite highlight when it comes to these two mappers.
```

> If you like energetic songs, then this is your perfect fit! JENNY ROM's "WWW.BLONDE GIRL" mapped by [Ascendance](https://osu.ppy.sh/users/2931883) brings the upbeat vibe with the song. With multiple guest difficulties, players can enjoy multiple interpretations of the song. On a side note, make sure to check out the background on Spec's Challenge!

Reviewer 5: I dont know if escaping the point in "WWW.BLONDE" will disable the automatic linking there, probably worth to try.

Author: This didn't help. Found another solution, might worth a try 👀

> The ultimate duo, consisting of [Ascendance](https://osu.ppy.sh/users/2931883) and [Fii](https://osu.ppy.sh/users/3922569), is back once again with their rendition of supercell's "My Dearest". Featuring an impressive storyboard by Storyboarder and PoNo, the map is quite the experience and manages to keep bringing fun and engaging patterns for its entire duration, making the map fly by. Make sure to check this map out, as it is a definite highlight when it comes to these two mappers.

Author: No one cares about storyboards. lol

> The ultimate duo, consisting of [Ascendance](https://osu.ppy.sh/users/2931883) and [Fii](https://osu.ppy.sh/users/3922569), is back once again with their rendition of Supercell's "My Dearest". Featuring an impressive storyboard by Storyboarder and PoNo, the map is quite the experience and manages to keep bringing fun and engaging patterns for its entire duration, making the map fly by. Make sure to check this map out, as it is a definite highlight when it comes to these two mappers.

Reviewer 3: Supercell's -> supercell's

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/catch/gensou.jpg)](http://osu.ppy.sh/beatmapsets/699439)

> Right before leaving for military service, [Spectator](https://osu.ppy.sh/users/702598) managed to give us another fantastic set, this time on a groovy song BUTAOTOME's "Gensou no Satellite". The highest difficulty manages to keep up fast and energetic patterns to keep up with the song. The rest of the set, including a Xetopia guest difficulty doesn't disappoint in giving something to offer to players of all skill levels.

Reviewer: link to Xetopia's profile

## <a name="mania" id="mania"></a>osu!mania

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/mania/emotional.jpg)](http://osu.ppy.sh/beatmapsets/800663)

[K a b i -](https://osu.ppy.sh/users/6209713) comes into the ranked mapping scene with an absolute banger of a song kamome sano's "\</emotional>", and a challenging long note heavy map that tests the player in release timing, consistency, and inverses. His addiction to mapping kamome sano pays off in this map in particular, his mapping style accompanying the music pretty well.

> [Critical_Star](https://osu.ppy.sh/users/3793196) follows suit in their 7K guest difficulty, following a similar long note based approach as seen in many of their previous ranked maps. From the more recent movement in the mania ranked section where mappers have been leaning towards long note based maps, emotional is one of the most solid of them, and greatly encapsulates the shift in the 4K mapping trends this season. We are definitely excited to see more long note based maps from K a b i -, Critical_Star, and other mappers in the future, and see the style of ranked maps shift gradually over time.

Reviewer: in the mania ranked section -> in the osu!mania ranked section

> [Critical_Star](https://osu.ppy.sh/users/3793196) follows suit in their 7K guest difficulty, following a similar long note based approach as seen in many of their previous ranked maps. From the more recent movement in the mania ranked section where mappers have been leaning towards long note based maps, emotional is one of the most solid of them, and greatly encapsulates the shift in the 4K mapping trends this season. We are definitely excited to see more long note based maps from K a b i -, Critical_Star, and other mappers in the future, and see the style of ranked maps shift gradually over time.

Reviewer: profile links at the end

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/mania/envidia.jpg)](http://osu.ppy.sh/beatmapsets/819153)

The autumn season of mania ranked seemed to mark a heavy interest in the long note side of things, and [Calcyon](https://osu.ppy.sh/users/2412652)'s addition in Nekomata Gekidan's "envidia" is certainly up there with the best of them this season. This marathon makes extremely nice use of varying LN patterns to mark key differences in sections of the map, using lighter layering, trills, inverse, and even some minijacks to clearly define the intensity of different sections. 

All of this is paired with difficult yet smooth transitions and patterning to make the experience all the more fun and rewarding for players. Envidia, through these choices, certainly proved itself to be a point of quality for the autumn season of mania.

> [![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/mania/smilling.jpg)](http://osu.ppy.sh/beatmapsets/810911)

Reviewer 2: Broken image.
```suggestion
[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/mania/smiling.jpg)](http://osu.ppy.sh/beatmapsets/810911)
```

> The autumn season of mania ranked seemed to mark a heavy interest in the long note side of things, and [Calcyon](https://osu.ppy.sh/users/2412652)'s addition in Nekomata Gekidan's "envidia" is certainly up there with the best of them this season. This marathon makes extremely nice use of varying LN patterns to mark key differences in sections of the map, using lighter layering, trills, inverse, and even some minijacks to clearly define the intensity of different sections. 

Reviewer: mania -> osu!mania

> All of this is paired with difficult yet smooth transitions and patterning to make the experience all the more fun and rewarding for players. Envidia, through these choices, certainly proved itself to be a point of quality for the autumn season of mania.

Reviewer: mania -> osu!mania

[Kawawa](https://osu.ppy.sh/users/4647754) comes back with a complete shift of his mapping style along with Lime's "Smiling". Kawawa interprets the melody line with complex long notes with its flow, mixed with harsh chord stream in between. As the song progresses, players will have to  go through fast streams and moreover read out the full long note chorus section. It is indeed a challenging map but also enjoyable for long note lovers. 

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/mania/genesis.jpg)](http://osu.ppy.sh/beatmapsets/779906)

> Hakuryu's "Genesis At Oasis (MOONLiGHT Mix)" mapped by [Alternater](https://osu.ppy.sh/users/4980256) featured in 4K MWC 2018, the top difficulty has already been well-known before it was made into this great full spread. Even though the patterns used contains mostly variations of basic jumpstreams, they closely fit with the music and therefore well represent the whole song. The SV use is also very comfortable and beneficially exaggerates the musical impact. 

Reviewer: MWC 2018 -> osu!mania World Cup 2018

> Hakuryu's "Genesis At Oasis (MOONLiGHT Mix)" mapped by [Alternater](https://osu.ppy.sh/users/4980256) featured in 4K MWC 2018, the top difficulty has already been well-known before it was made into this great full spread. Even though the patterns used contains mostly variations of basic jumpstreams, they closely fit with the music and therefore well represent the whole song. The SV use is also very comfortable and beneficially exaggerates the musical impact. 

Reviewer: patterns used contains -> patterns used contain*

> Hakuryu's "Genesis At Oasis (MOONLiGHT Mix)" mapped by [Alternater](https://osu.ppy.sh/users/4980256) featured in 4K MWC 2018, the top difficulty has already been well-known before it was made into this great full spread. Even though the patterns used contains mostly variations of basic jumpstreams, they closely fit with the music and therefore well represent the whole song. The SV use is also very comfortable and beneficially exaggerates the musical impact. 

Reviewer: The SV use -> The slider velocity manipulation

> [-MysticEyes](https://osu.ppy.sh/users/6253266), [Unpredictable](https://osu.ppy.sh/users/7560872), and [qqqant](https://osu.ppy.sh/users/8976576) also thoughtfully contribute to the set by mapping solid lower difficulties for this set. Overall, this set expresses the music perfectly and deserves a place in Spotlight!

Reviewer: thoughtfully contributeD*

> [-MysticEyes](https://osu.ppy.sh/users/6253266), [Unpredictable](https://osu.ppy.sh/users/7560872), and [qqqant](https://osu.ppy.sh/users/8976576) also thoughtfully contribute to the set by mapping solid lower difficulties for this set. Overall, this set expresses the music perfectly and deserves a place in Spotlight!

Reviewer: in Spotlight! -> in this season's Spotlights!

[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/mania/dream-catcher.jpg)](http://osu.ppy.sh/beatmapsets/696380)

> Here [Salty Mermaid](https://osu.ppy.sh/users/9037365) makes his debut into the mapping scene with his song, MYLK's "Dream Catcher". The most notable and memorable of this map is by far the kiai with its fast paced patterns implementing both jacks and trills in a way that heavily compliment the song and makes the song that much more enjoyable upon revisiting.

Reviewer: heavily compliment the song and make* the song although it's redundant, remove the first "the song"

---

> The spreadsheet for this seasons selection is available and fully viewable [here](https://goo.gl/2waBVj). Feel free to check it out!

Author: I prefer shortened and I think it is not editable??? 👀 Or at least it shouldn't be. I locked it down.

> The spreadsheet for this seasons selection is available and fully viewable [here](https://goo.gl/2waBVj). Feel free to check it out!

Author: I can't check tho since I have all permission

> We hope you enjoy our selection for this season and look forward to what we will bring to you in the future. We have a lot of plan for the next year! If you want to join to us bringing new stuff to life, you can hop onto our Discord [server](http://discord.gg/Z28ZtUf) or apply as a member [here](https://osu.spotlights.team).

Reviewer: a lot of plan -> a lot of planS

> We hope you enjoy our selection for this season and look forward to what we will bring to you in the future. We have a lot of plan for the next year! If you want to join to us bringing new stuff to life, you can hop onto our Discord [server](http://discord.gg/Z28ZtUf) or apply as a member [here](https://osu.spotlights.team).

Reviewer: place Discord server as hyperlink, not only server
